scripts/main/album.js

```diff
@@ -775,6 +783,12 @@ album.setPublic = function (albumID, e) {
 			}
 		});
 
+		if (album.json.nsfw === "1") {
+			$('.basicModal .switch input[name="nsfw"]').prop("checked", true);
```

Comment:
I'm guessing we can't just
```js
$('.basicModal .switch input[name="nsfw"]').prop("checked", album.json.nsfw === "1");
```
?

Reply:
I didn't think about it, just copy pasted the pattern around.

Reply:
I think it looks cleaner, but I don't really care.